### DISCUSSION TITLE
How to get a Material of namespace id including generic ones like minecraft:planks

### DISCUSSION THREAD

**Discussion by sarpedondev** [Minestom/Minestom#1735] [Help] [answered]
Title. Basically is there a function to return a Material of a namespace ID and if the namespace ID is a generic one like minecraft:planks instead of minecraft:oak_planks for example, it would have to tell me. The data is in block_tags.json, I'm just not sure how to use it, thanks!

**Comment by mworzala** [comment, marked as answer]
`minecraft:planks` is one of many tags that `minecraft:oak_planks` is part of, its not a 1:1. You can access tags via `TagManager`, though it is a bit half baked right now, for example there is not a reverse index eg "block->tags it has". Though you could construct this yourself from the tag data.
